common/persistence/sql/sqlplugin/visibility.go

```diff
@@ -54,6 +55,7 @@ type (
 	VisibilityRow struct {
 		NamespaceID          string
 		RunID                string
+		Version 		     int64
```

Comment:
Can you maybe separate this from the rest, and add comments explaining this field?
```
VisibilityRow struct {
	// existing fields

	// explain version
	Version int64
}
```

Reply:
I choose the position to match the order of the columns in the table (if you just use create statements) 

but, the more I think about this the more I like it. Because Go reflects fields in the order of declaration, we would remove the need for the hacky reordering and have no SQL DB type specific code in the main plugin. I'm going to do that.

Reply:
Done